apps/web/src/pages/meeting-rooms/index.tsx

```
@@ -1,14 +1,117 @@
+import { Resource } from "@repo/constants/type";
+import { getGroupedResourceListBySubtype } from "@repo/lib/api/resourceUtils";
+import Drawer from "@src/components/commons/Drawer";
+import ErrorBoundary from "@src/components/commons/ErrorBoundary";
+import { isOpenDrawerAtom } from "@src/store/isOpenDrawerAtom";
+import { useQueryClient } from "@tanstack/react-query";
+import { AnimatePresence } from "framer-motion";
+import { useAtom, useAtomValue, useSetAtom } from "jotai";
+import { Suspense, useMemo } from "react";
+
+import Error from "../seats/Error";
```

Biome

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)